File: pset6/Sentimental/cash.py
from cs50 import get_float
from sys import exit

while True:
	change_o = get_float("Change owed: ")

	if change_o >= 0:
		break


coins = change_o * 100

round(coins, 1)

quarters = coins // 25
dimes = (coins % 25) // 10
nickels = ((coins % 25) % 10) // 5
pennies = ((coins % 25) % 10) % 5

total = quarters + dimes + nickels + pennies

print(int(total))

File: pset1/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main()
{
    int coins;

    do
    {
        float change_o = get_float("Change owed: ");
        coins = round(change_o * 100);
    }
    while (coins <= 0);

    int quarters = coins / 25;
    int dimes = (coins % 25) / 10;
    int nickels = ((coins % 25) % 10) / 5;
    int pennies = ((coins % 25) % 10) % 5;

    printf("%d\n", quarters + dimes + nickels + pennies);
}
